Xamarin.Forms.Platform.WinRT/SwitchRenderer.cs

```diff
@@ -48,5 +58,16 @@ void OnNativeToggled(object sender, RoutedEventArgs routedEventArgs)
 		{
 			((IElementController)Element).SetValueFromRenderer(Switch.IsToggledProperty, Control.IsOn);
 		}
+
+		void UpdateFlowDirection()
+		{
+			if (VisualElementController == null || Control == null)
+				return;
+
+			if (VisualElementController.EffectiveFlowDirection.IsRightToLeft())
+				Control.FlowDirection = WFlowDirection.RightToLeft;
+			else if (VisualElementController.EffectiveFlowDirection.IsLeftToRight())
+				Control.FlowDirection = WFlowDirection.LeftToRight;
+		}
```

Comment:
It's not ur fault but it's a shame our conventions lead us to repeat this code in so many places...

Reply:
The conventions don't force the repetition of the stuff inside `UpdateFlowDirection` - all of that could be moved into an extension method.